data/themes/default/index.html

```diff
@@ -336,6 +336,25 @@
     }
 });
 
+function CreateSongString(song, showEdition) {
```

Comment:
The whole `index.html` should be refactored imho. When i have time i might turn it into a nice project instead of 1 file ;) But that was not the case within this request hahah